### DISCUSSION TITLE
Matomo Analytics?

### DISCUSSION THREAD

**Discussion by ModischFabrications** [arthelokyo/astrowind#355] [Ideas]
The new data privacy laws (GDPR) in the EU make usage of google analytics difficult, if not impossible. 
Getting another analytics integration would be nice, one of the bigger alternatives is https://matomo.org/. 
They allow self-hosting, see <https://matomo.org/faq/on-premise/installing-matomo/>

Feel free to close if not relevant.

**Comment by widgeter** [comment]
Hi @ModischFabrications 

I've been reviewing [Matomo](https://matomo.org/) and the new GDPR laws. As I see, Matomo comes from Piwik, which I had used in the past.

I think what you're saying makes a lot of sense.

I'm going to move this issue to Discussion and we'll plan to work on it.

Thank you so much.
